README.md

```diff
@@ -209,11 +210,25 @@ export default new Vuex.Store({
 })
 ```
 
-#### Namespaced vuex modules
+#### Emitting Events  
+
+Events can be sent to the Socket.IO server by calling 
+```js
+this._vm.$socket.client.emit
```

Comment:
Use it like so: `this._vm.$socket.client.emit` 
no need for new lines and syntax highlight here

I'll merge after that change

Reply:
Awesome, thanks!